builds/e2e/isa-95-smoke-test.yaml

```diff
@@ -203,6 +203,10 @@ stages:
         parameters:
           agentName: $(deviceLvl4AgentName)
           lvl: 4
+      - template: templates/nested-isa95-lock.yaml      
+        parameters:
+          agentName: "otproxy"
```

Comment:
Is this supposed to be hardcoded?

Reply:
Yes but I can do better than this. I moved it at the top with other hardcoded name.